next/docs/iota-sdk/docs/how-tos/alias-wallet/request-funds.mdx

```diff
@@ -19,19 +19,170 @@ import TabItem from '@theme/TabItem';
 import TipFaucet from '../../_admonitions/_tip-faucet.md';
 import AliasNFT from '../../_admonitions/_alias_nft_wallet.md';
 
-Alias and NFT outputs have their own address and therefore can hold their own base token or any other native asset.
+Alias addresses play a significant role in decentralized systems, particularly in managing and identifying various assets.
+Unlike traditional addresses associated with specific user accounts, Alias addresses are specifically assigned to Alias
+and NFT outputs. These outputs have their own unique addresses, enabling them to hold their respective base tokens or
+any other native assets. This means that you could transfer ownership of all the assets controlled by the Alias address
+by transferring its Alias output.
```

Comment:
```suggestion
Alias and NFT addresses play a significant role in decentralized systems, particularly in managing and identifying various assets.
Unlike traditional addresses associated with specific user accounts, those addresses are specifically assigned to Alias
and NFT outputs. These outputs have their own unique addresses, enabling them to hold their respective base tokens or
any other native assets. This means that you could transfer ownership of all the assets controlled by the such an address
by transferring its output.
```

A quick fix could be this @luca-moser. Otherwise we need to wait for @lucas-tortora 😆